src/legacy/ui/public/management/saved_objects_management/index.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export { SavedObjectsManagementActionRegistry } from './saved_objects_management_action_registry';
+export {
+  SavedObjectsManagementAction,
+  SavedObjectRecord,
+  SavedObjectReference,
+} from './saved_objects_management_action';
+export {
+  processImportResponse,
+  ProcessedImportResponse,
+} from '../../../../core_plugins/kibana/public/management/sections/objects/lib/process_import_response';
```

Comment:
Ah, I thought this was acceptable because we do the same thing from [`src/legacy/ui/public/management/index.js`](https://github.com/elastic/kibana/blob/2ce51a5be583a0a58c38ed1426efdbb9e5f19ecc/src/legacy/ui/public/management/index.js#L20-L28) today to share functionality between OSS and X-Pack